src/deepsparse/server/config.py

```diff
@@ -48,6 +48,16 @@ class ImageSizesConfig(BaseModel):
     )
 
 
+class TargetLoggingConfig(BaseModel):
+    target: str = Field(
+        description="Name of the target to apply the metric functions to."
+    )
+    functions: List[Dict[str, Any]] = Field(
```

Comment:
optional nit to do later after this PR (non blocking IMO)
```suggestion
    functions: List[Dict[str, FunctionConfig]] = Field(
```
so we can get validation on FunctionConfig, but FunctionLogger would be unchanged (i.e. the build_logger would pass field from FUnctionConfig to FunctionLogger)

Reply:
yes - the `FunctionConfig` is really the piece that needs validation. TargetLoggingConfig is less necessary as it's essentially a key value pair with target: functions. Prefer to change data_logging in server to be type `Dict[str, List[FunctionConfig]]`

so it would be 
```
data_logging:
  identifier_1:
      - function_config...
```